tests/validators/test_sanity_checks.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+
+from electricitymap.contrib.validators.sanity_checks import (
+    validate_positive_production,
+    validate_production_one_non_nan_value,
+)
+
+from .lib.load_fixture import load_fixture
+
+
+def test_validate_positive_production():
+    events = load_fixture("production_negative_values")
+    res = validate_positive_production(events)
+    assert res["2022-01-01 00:00:00+00:00"] == 1
```

Comment:
could simplify by checking that the column is [1, 0, 1]

Reply:
👍 I'll keep it in mind for other tests. I think for these I'll keep the multiple asserts as they are a bit more readable.